Extension/src/Debugger/configurationProvider.ts

```diff
@@ -593,13 +593,100 @@ export class DebugConfigurationProvider implements vscode.DebugConfigurationProv
         return false;
     }
 
+    private getLaunchJsonPath(): string | undefined {
+        return util.getJsonPath("launch.json");
+    }
+
+    public getRawLaunchJson(): Promise<any> {
+        const path: string | undefined = this.getLaunchJsonPath();
+        return util.getRawJson(path);
+    }
+
+    public async writeDebugConfig(config: vscode.DebugConfiguration, folder?: vscode.WorkspaceFolder): Promise<void> {
+        const settings: OtherSettings = new OtherSettings();
```

Comment:
Move `const settings: OtherSettings = new OtherSettings();` to line 364 just before it is used?